tests/app/inbound_number/test_rest.py

```diff
@@ -47,14 +47,36 @@ def test_gets_inbound_numbers(self, admin_request, mocker):
 
 class TestSetInboundNumberOff:
     def test_sets_inbound_number_active_flag_off(self, admin_request, mocker):
-        dao_set_inbound_number_active_flag = mocker.patch('app.inbound_number.rest.dao_set_inbound_number_active_flag')
+        """
+        Test that the endpoint sets the active flag of an inbound number to False.
+        """
+        dao_set_inbound_number_active_flag = mocker.patch(
+            'app.inbound_number.rest.dao_set_inbound_number_active_flag', return_value=None
+        )
 
         inbound_number_id = uuid.uuid4()
         admin_request.post(
             'inbound_number.post_set_inbound_number_off', _expected_status=204, inbound_number_id=inbound_number_id
         )
         dao_set_inbound_number_active_flag.assert_called_with(inbound_number_id, active=False)
```

Comment:
This test should not mock anything.  It should create an inbound number, check that it is active, make the request to the endpoint, refresh the inbound number, and check that it is inactive.  It should not assert that anything was called.